http-server/src/main/java/io/strimzi/admin/Main.java

```diff
@@ -42,4 +53,36 @@ public static void main(final String[] args) {
 
         return promise.future();
     }
+
+    private static Map envVarsToAdminClientConfig() throws Exception {
+        Map envConfig = System.getenv().entrySet().stream().filter(entry -> entry.getKey().startsWith(PREFIX))
+                .collect(Collectors.toMap(entry -> entry.getKey().substring(PREFIX.length()), Map.Entry::getValue));
+
+        Map<String, String> adminClientConfig = new HashMap();
+        if (envConfig.get("BOOTSTRAP_SERVERS") == null) {
+            throw new Exception("Bootstrap address has to be specified");
+        }
+        adminClientConfig.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, envConfig.get("BOOTSTRAP_SERVERS").toString());
+
+        if (envConfig.get("SSL_KEYSTORE_LOCATION") != null && envConfig.get("SSL_KEYSTORE_PASSWORD") != null) {
+            adminClientConfig.put(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG, envConfig.get("SSL_KEYSTORE_LOCATION").toString());
+            adminClientConfig.put(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG, envConfig.get("SSL_KEYSTORE_PASSWORD").toString());
+            adminClientConfig.put(SslConfigs.SSL_KEYSTORE_TYPE_CONFIG, "PKCS12");
+        }
+
+        if (envConfig.get("SSL_TRUSTSTORE_LOCATION") != null && envConfig.get("SSL_TRUSTSTORE_PASSWORD") != null) {
+            adminClientConfig.put(AdminClientConfig.SECURITY_PROTOCOL_CONFIG, "SSL");
+            adminClientConfig.put(SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG, envConfig.get("SSL_TRUSTSTORE_LOCATION").toString());
+            adminClientConfig.put(SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG, envConfig.get("SSL_TRUSTSTORE_PASSWORD").toString());
+            adminClientConfig.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG, "PKCS12");
+        }
+
+        adminClientConfig.put(AdminClientConfig.METADATA_MAX_AGE_CONFIG, "30000");
+        adminClientConfig.put(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG, "10000");
+        adminClientConfig.put(AdminClientConfig.RETRIES_CONFIG, "3");
+        adminClientConfig.put(AdminClientConfig.DEFAULT_API_TIMEOUT_MS_CONFIG, "40000");
```

Comment:
from where these values are coming?

Reply:
We use these in operators

Reply:
and what was the rationale behind them? I don't remember tbh, so don't know if the same make sense for admin server. Let's stay with them for now but we have to investigate.